ethereum/eth/src/jmh/java/org/hyperledger/besu/ethereum/eth/sync/worldstate/WorldStateDownloaderBenchmark.java

```diff
@@ -120,7 +121,8 @@ public void setUpUnchangedState() {
             syncConfig.getWorldStateMaxRequestsWithoutProgress(),
             syncConfig.getWorldStateMinMillisBeforeStalling(),
             Clock.fixed(Instant.ofEpochSecond(1000), ZoneOffset.UTC),
-            metricsSystem);
+            metricsSystem,
+            SyncDurationMetrics.NO_METRICS_SYNC_DURATION_METRICS);
```

Comment:
there seems to be quite a few of these NO_METRICS_SYNC_DURATION_METRICS, it might be useful to have an overloaded constructor that delegates to the main constructor with NO_METRICS...

Reply:
Agree.